databricks/koalas/frame.py

```diff
@@ -691,8 +727,86 @@ def loc(self):
     def copy(self):
         return DataFrame(self._sdf, self._metadata.copy())
 
-    @derived_from(pd.DataFrame)
     def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
+        """
+        Remove missing values.
+
+        Parameters
+        ----------
+        axis : {0 or 'index'}, default 0
+            Determine rows which contain missing values are removed.
+            * 0, or 'index' : Drop rows which contain missing values.
+            .. dropna currently only works for axis=0 or axis='index'
+               axis=1 is yet to be implemented.
+        how : {'any', 'all'}, default 'any'
+            Determine if row or column is removed from DataFrame, when we have
+            at least one NA or all NA.
+            * 'any' : If any NA values are present, drop that row or column.
+            * 'all' : If all values are NA, drop that row or column.
+        thresh : int, optional
+            Require that many non-NA values.
+        subset : array-like, optional
+            Labels along other axis to consider, e.g. if you are dropping rows
+            these would be a list of columns to include.
+        inplace : bool, default False
+            If True, do operation inplace and return None.
+
+        Returns
+        -------
+        DataFrame
+            DataFrame with NA entries dropped from it.
+
+        See Also
+        --------
+        DataFrame.drop : Drop specified labels from columns.
+        DataFrame.isnull: Indicate missing values.
+        DataFrame.notnull : Indicate existing (non-missing) values.
```

Comment:
We don't have `notnull()`.
You can replace it by `notna()`.

Reply:
We do have notnull, don't we? https://github.com/databricks/koalas/blob/master/databricks/koalas/frame.py#L535

Reply:
Line 571 is `notnull()` in frame.py.
Please correct me if I am making any mistake here.

Reply:
I saw the documentation. You re right